hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializer.java

```diff
@@ -87,6 +91,11 @@ public Collection<Class> getCompactSerializableClasses() {
         return classToRegistrationMap.keySet();
     }
 
+    public boolean canBeSerializedAsCompact(Class<?> clazz) {
+        return serializationService.serializerForClass(clazz, false) instanceof CompactStreamSerializerAdapter
+                || serializationService.serializerForClass(clazz, false) instanceof CompactStreamSerializerAdapter;
```

Comment:
`serializationService.serializerForClass(clazz, false) instanceof CompactStreamSerializerAdapter` was repeated twice.

Reply:
Yeah, I was gonna add an instance of check with another class, saw that it is unnecessary, but forget to update it. Will do it now